OSDOCS-19946: Reorganizing securing routes modules (#112654)
openshift/openshift-docs: networking/ingress_load_balancing/routes/creating-advanced-routes.adoc

> :_mod-docs-content-type: ASSEMBLY

Reviewer: When managing certificates via `Ingress` objects, users should be warned that the Ingress Controller maintains a one-way sync. Any manual 'quick fixes' applied directly to the generated `Route`'s TLS configuration will be silently overwritten the next time the parent `Ingress` object is updated or reconciled. This is a common source of confusion in GitOps-managed clusters.

Author: Excellent point, I've added a Warning box

:_mod-docs-content-type: ASSEMBLY
[id="creating-advanced-routes"]
= Securing routes via ingress objects
include::_attributes/common-attributes.adoc[]
:context: creating-advanced-routes

toc::[]

[role="_abstract"]

> You can secure your application traffic by managing certificates directly through ingress objects. This includes creating routes using the destination CA certificate in an ingress annotation or utilizing the default certificate.

Reviewer 2: ```suggestion
You can secure your application traffic by managing certificates directly through ingress objects. This includes creating routes using the destination CA certificate in an ingress annotation or using the default certificate.
```

Author: Got it, thanks!

[WARNING]
====

> The Ingress Controller maintains a one-way sync for certificates managed through ingress objects. Do not manually apply changes directly to the generated route's TLS configuration. Any manual modifications will be silently overwritten the next time the parent ingress object is updated or reconciled. This is particularly important to note if you operate a GitOps-managed cluster.

Reviewer 2: ```suggestion
The Ingress Controller maintains a one-way sync for certificates managed through ingress objects. Do not manually apply changes directly to the generated route's TLS configuration. Any manual modifications are silently overwritten the next time the parent ingress object is updated or reconciled. This is particularly important to note if you operate a GitOps-managed cluster.
```

Author: done, thanks
